### PR TITLE
[#9666] Missing translations on "Lock app with fingerprint", "Save wi…

### LINKED ISSUE #9666: Missing translations on "Lock app with fingerprint", "Save with password" popups
# Bug Report

## Problem
photo_2019-12-16 15 11 47
photo_2019-12-16 15 12 02

#### Expected behavior
translation is added 

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Create account with keycard
- Go to Settings > Privacy and Security > Enable app with fingerprint
- Logout
- Tap on "Don't ask for keycard to sign in"

### Additional Information

- Status version: nightly 16/12/2019
- Operating System: Android

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #9666

### CODE REVIEW

File: src/status_im/ui/screens/biometric/views.cljs [hunk 1]
```diff
@@ -72,11 +73,9 @@
...
     [biometric-popover
-     {:title-label     :t/biometric-secure-with
-      :ok-button-label :t/biometric-enable-button
-      :on-confirm      :biometric/enable
-
-      :description-label
-      (if keycard-account?
-        (i18n/label :t/biometric-enable-keycard)
-        (i18n/label :t/biometric-enable))}]))
+     {:title-label       :t/biometric-secure-with
```
Comment (yenda) [inline src/status_im/ui/screens/biometric/views.cljs:76]: it doesn't look like anything changed here

Comment (yenda) [reply src/status_im/ui/screens/biometric/views.cljs:76]: there is still some fixes needed as done here https://github.com/status-im/status-react/pull/9716/files

Comment (flexsurfer) [reply src/status_im/ui/screens/biometric/views.cljs:76]: there is no `i18n/label` for :description-label

Comment (flexsurfer) [reply src/status_im/ui/screens/biometric/views.cljs:76]: 9716 has wrong code, this PR fixed it

**Review by yenda — APPROVED** [review]

**Comment by churik** [comment]
Tested on Keycard and ordinary accounts

**Pull request merged**